common/src/main/kotlin/com/cosmotech/common/utils/SecurityUtils.kt

```diff
@@ -55,6 +55,14 @@ fun getCurrentAccountIdentifier(configuration: CsmPlatformProperties): String {
   }
 }
 
+fun getCurrentAccountGroups(configuration: CsmPlatformProperties): List<String> {
+  val authentication = getCurrentAuthentication()
+  val jwt = (authentication as JwtAuthenticationToken).token.tokenValue
+  val jwtClaimsSet = JWTParser.parse(jwt).jwtClaimsSet
+  return jwtClaimsSet.getListClaim(configuration.authorization.groupJwtClaim).toList()
```

Comment:
Could you please rewrite this using `getCurrentAuthenticatedRoles` as template?
`jwtClaimsSet.getListClaim(configuration.authorization.groupJwtClaim).toList() as List<String>`:
- this line can lead to NullPointerException
- use `getStringListClaim` instead of `getListClaim` and remove `.toList()` call and unecessary cast `as List<String>`
- return an empty list instead of a null